### PR TITLE
Update for TokenTree move in nightly.

### PR DESCRIPTION
**Pull request by birkenfeld**

### CODE REVIEW

**Comment by LeoTestard** [comment]
Thanks for the PR! I should have done this since weeks ago...
Looks like there is another error in the Travis build. Can you fix it as well? It should be straightforward.

File: codegen/lib.rs [hunk 1]
```diff
@@ -11,7 +11,12 @@
 extern crate bit_set;
 extern crate fsa;
 
-use syntax::ast::{Ident, TokenTree};
+#[cfg(not(feature = "with-syntex"))]
+use syntax::tokenstream::TokenTree;
+#[cfg(feature = "with-syntex")]
+use syntax::ast::TokenTree;
```
Comment (LeoTestard) [inline codegen/lib.rs]: Also, I find it better style to put the code on the same line as the attribute when it can fit. :P
But it's not too important...

**Comment by birkenfeld** [comment]
Fixed.

**Comment by LeoTestard** [comment]
👍

**Pull request merged**